misc/task-server-endpoint-validator/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
```

Comment:
If you use an image like `busybox` which already includes `wget` or an image like `amazonlinux:2` which already includes `curl`, you can avoid having to construct a new image and instead can just override the command.

Reply:
awesome! will change it to al2 image